### ISSUE TITLE
Settings changes not are not showing up in graphical Dconf editor, nor are they keeping their value

### ISSUE THREAD

**Issue by sudoman** [jistr/ansible-gsetting#3]
If I use your module then I see that according to a `gsettings get` that the value appears set. However the graphical Dconf editor doesn't show the change. Eventually the value reverts to its previous value.

I've tried calling `gsettings set` from a remote shell, setting the `$DISPLAY` variable as requested. This method works.

I have the same problem with your `ansible-dconf` repo.

I'm using Trisquel 8, which is based on Ubuntu 16.04.

Thanks. :-)

**Comment by sudoman** [comment]
There are at least two issues that come into play when the remote user is logged into a graphical environment with a running dbus session:

1) Don't use this module when dconf-editor is running. If a user is logged in, make sure this application isn't running. It appears to cache settings and re-apply them if it doesn't hear about a change coming from the primary dbus process.

2) Some other issue I haven't worked out yet. It seems like this command works while ssh'd into a user, sudo'd up and su'd into the user:

    export `/usr/bin/dbus-launch` ; /usr/bin/gsettings set org.mate.screensaver idle-activation-enabled false ; kill $DBUS_SESSION_BUS_PID &> /dev/null

I've also experimented with adding sleep statements to the two commands.

However if we pop the shell stack back to the root shell, then running `su - user -c ...` doesn't seem to work. Just querying the value appears to change it, unless there is some other process that's coincidentally making the change.

**Comment by tyleha** [comment]
I also have found that while `gsettings get` will report the right value for a bit with this method, the changes do not immediately take effect and in fact are lost on user logout/log back in. 

The solution that immediately and permanently has worked for me is https://askubuntu.com/a/743024/358498 which comes down to:

```
PID=$(pgrep gnome-session)
export $(grep -z DBUS_SESSION_BUS_ADDRESS /proc/$PID/environ)
```

thus setting the `DBUS_SESSION_BUS_ADDRESS` variable. Afterwards, a simple call to `gsettings set org.gnome.desktop.session idle-delay 0`, for instance, works immediately.

**Comment by tyleha** [comment]
Further testing shows that the presence of an existing gnome-session might be the reason the current method used in this playbook doesn't persist. It seems to work OK if no user is logged in on the system and thus there is no gnome-session.

**Comment by Yannik** [comment]
I've provided a fix for this in my PR #6. Thanks @tyleha for your input!
